css-view-transitions-1/Overview.bs

```diff
@@ -627,7 +627,10 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 	a [=pseudo-element root=].
 	Its [=originating element=] is the document's [=document element=].
 
-	Its [=containing block=] is the [=snapshot root=].
+	Its [=containing block=] is the [=snapshot containing block=].
+
+	The [=snapshot containing block=] is considered to be an [=absolute positioning containing block=]
```

Comment:
@bokand @bfgeek do you mind reviewing this? This is the spec text which defines the containing block for view-transition pseudo-elements. If `::view-transition` is set to `position: static` by the author then it, and any descendant which would've used `::view-transition` as its containing block, should instead use the snapshot containing block.

Reply:
Would it make sense to put this bit ("the SCB is considered to be an APCB and a FPCB") down with the definition of Snapshot Containing Block?

Also, above:

```
<div class=note>This element provides a containing block for all ''::view-transition-group()'' pseudo-elements.</div>
```

This depends on the style given to `::view-transition` - perhaps it should instead say "This element serves as a parent for all `::view-transition-group()` pseudo-elements"?

Reply:
Fixed, thanks!